src/test/mocks/archiveLoader.ts

```diff
@@ -1,30 +1,31 @@
+// tslint:disable: max-line-length
 import fs from 'fs';
 import cloneDeep from 'lodash/fp/cloneDeep';
 import path from 'path';
 import { ArchiveBook, ArchivePage } from '../../app/content/types';
 
 export const book = JSON.parse(
-  fs.readFileSync(path.resolve(__dirname, '../fixtures/contents/testbook1-shortid.json'), 'utf8')
+  fs.readFileSync(path.resolve(__dirname, '../fixtures/apps/archive/20210224.204120/contents/testbook1-shortid.json'), 'utf8')
 ) as ArchiveBook;
 
 export const page = JSON.parse(
-  fs.readFileSync(path.resolve(__dirname, '../fixtures/contents/testbook1-shortid:testpage1-shortid.json'), 'utf8')
+  fs.readFileSync(path.resolve(__dirname, '../fixtures/apps/archive/20210224.204120/contents/testbook1-shortid:testpage1-shortid.json'), 'utf8')
```

Comment:
you might want to use a more placeholdery name for the test archive url, like `apps/archive/codeversion/contents` or something, otherwise this version is going to be saved for the rest of time in the tests or we'll have to update it when we update the prod one even if we don't really have to